Rogue-Ape-Studios/Crusader
# ✨ Add support for time-based level switching

## Content

Added support for triggering `OnLevelComplete` with a timer.

## Changes


### Updated
`Assets/Scripts/Spawn/SpawnManager.cs` : Was updated with a timer for supporting the event.

## Testing

The feature / Bugfix can be testing by following these steps:

1. Set the timer to 10 seconds.
2. Wait 10 seconds.
3. The `SpawnManager` Component should be disabled.
4. Alternatively, write a quick debug statement in `HandleWaveComplete`.
